scripts/us_eia/opendata/README.md

```diff
@@ -22,9 +16,45 @@ This dataset is available for public use, license is available at https://www.ei
 
 ### Import procedure
 
-- Download data 
+- Download data and will process the data
     ```bash
-    python3 download_bulk.py
+    python3 process.py --dataset=TOTAL
+    python3 process.py --dataset=INTL
+    python3 process.py --dataset=ELEC
+    python3 process.py --dataset=NG
+    python3 process.py --dataset=PET
+    python3 process.py --dataset=SEDS
+    python3 process.py --dataset=NUC_STATUS
+
     ```
 
-- Run the [processor](process/README.md)
\ No newline at end of file
+- Run the [processor](process/README.md)
+
+### Downloading and Processing Data
+
+
+    If you want to perform "only download", run the below command:
+
+        python3 process.py --dataset=INTL --mode=download
+        python3 process.py --dataset=ELEC --mode=download
+        python3 process.py --dataset=PET --mode=download
+        python3 process.py --dataset=NG --mode=download
+        python3 process.py --dataset=SEDS --mode=download
+        python3 process.py --dataset=NUC_STATUS --mode=download
+        python3 process.py --dataset=TOTAL --mode=download
+
+
+
+   If you want to perform "only process", run the below command:
+
+   Running this command generates input_fles and csv, mcf, tmcf, svg.mcf files
+
+        python3 process.py --dataset=INTL --mode=process
+        python3 process.py --dataset=ELEC --mode=process
+        python3 process.py --dataset=PET --mode=process
+        python3 process.py --dataset=NG --mode=process
+        python3 process.py --dataset=SEDS --mode=process
+        python3 process.py --dataset=NUC_STATUS --mode=process
+        python3 process.py --dataset=TOTAL --mode=process
+        
+   
```

Comment:
Please add a section to explain the behavior of the script when no mode is specified. This should include details on the default actions, which may involve both downloading and processing the data.